netlify.toml

```diff
@@ -17,3 +17,23 @@ to = "/docs/what-is-kudo.html#under-the-hood"
 [[redirects]]
 from = "/blog/blog-2020-07-building-your-first-operator-2.html"
 to = "/blog/blog-2020-07-13-building-your-first-operator-2.html"
+
+[[redirects]]
+from = "/docs/cli.html#setup-the-kudo-kubectl-plugin"
+to = "/docs/cli/installation.html"
+
+[[redirects]]
+from = "/docs/cli.html#commands"
+to = "/docs/cli/commands.html"
+
+[[redirects]]
+from = "/docs/cli.html#flags"
+to = "/docs/cli/commands.html"
+
+[[redirects]]
+from = "/docs/cli.html#examples"
+to = "/docs/cli/examples.html"
```

Comment:
Note that these don't work. `#` is a client-side concept, it does not even get sent to the server.
See for example https://deploy-preview-291--kudo-www.netlify.app/docs/cli.html#commands - still lands at `.../installation.html`

I see two ways around this:
1. leave a stub `content/docs/cli.md` behind, which would contain an 'update your links' note, section names and links to the new places. It's ugly, and not sure if vuepress will even let us have `cli.md` file and `cli` directory together
2. add anchors to `installation.md` somewhere at the bottom, with a single sentence linking to `examples.html` and `commands.html`

Either way these redirects for `#` URIs need to be removed.

Reply:
I've added a `cli/cli.md` that contains the old anchors and links to the correct subdocuments and added redirect from `cli.html` to cli/cli.html`, hope this works.